tests/storage/__init__.py

```diff
@@ -1,6 +1,9 @@
 from .storage import (
     cold_migration_then_come_back,
+    install_randstream,
     live_storage_migration_then_come_back,
+    operation_on_vdi,
     try_to_create_sr_with_missing_device,
     vdi_is_open,
+    wait_for_vdi_coalesce,
 )
```

Comment:
This remark might feel a bit off-topic, but having to re-export all those funcs seems awkward.
This `tests.storage` module seems inconsistent with the usual practice to put shared code inside `lib`, we might want to move that module instead of having to maintain this `__init__.py`?

Reply:
Yes, that's unusual. I'll move it in another PR.